packages/design-system/src/components/SimpleMenu/SimpleMenu.tsx

```diff
@@ -23,51 +24,57 @@ interface SimpleMenuProps
   onReachEnd?: (entry: IntersectionObserverEntry) => void;
 }
 
-const SimpleMenu = ({ children, onOpen, onClose, popoverPlacement, onReachEnd, ...props }: SimpleMenuProps) => {
-  /**
-   * Used for the intersection observer
-   */
-  const contentRef = React.useRef<HTMLDivElement>(null);
+const SimpleMenu = React.forwardRef<HTMLButtonElement, SimpleMenuProps>(
+  ({ children, onOpen, onClose, popoverPlacement, onReachEnd, ...props }, forwardedRef) => {
+    const triggerRef = React.useRef<HTMLButtonElement>(null);
```

Comment:
Where is triggerRef used?

Reply:
Its passed as a composedRef to MenuTrigger